README.md

```diff
@@ -359,11 +404,36 @@ const { result, error } = await deepgram.listen.prerecorded.transcribeFileCallba
 );
 ```
 
-[See our API reference for more info](https://developers.deepgram.com/reference/pre-recorded).
+[See our API reference for more info](https://developers.deepgram.com/reference/speech-to-text-api/listen).
+
+[See our Example for more info](). @TODO check if we need an example
```

Comment:
@naomi-lgbt  @lukeocodes  Do you want me to add an example here? We currently don't have one for this case.